sdk/python/kfp/deprecated/components_tests/test_python_op.py

```diff
@@ -254,6 +254,21 @@ def add_two_numbers_indented(a: float, b: float) -> float:
 
         self.helper_test_2_in_1_out_component_using_local_call(func, op)
 
+    def test_func_to_container_multiprocessing_spawn(self):
+        def add_two_numbers_mp(a: float, b: float) -> float:
+            import operator
+            import multiprocessing as mp
+            from concurrent.futures import ProcessPoolExecutor
+
+            with ProcessPoolExecutor(1, mp.get_context("spawn")) as pool:
+                future = pool.submit(operator.add, a, b)
+                return future.result()
+
+        func = add_two_numbers_mp
+        op = comp.func_to_container_op(func, use_code_pickling=True)
```

Comment:
nit: It's possible I'm missing something here, but I don't think `use_code_pickling` is required for this test. If this is correct, please remove since its inclusions suggests it might be related to the content of the test.


```suggestion
        op = comp.func_to_container_op(func)
```